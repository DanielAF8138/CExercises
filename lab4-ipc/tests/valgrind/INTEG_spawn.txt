==3308148== Memcheck, a memory error detector
==3308148== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==3308148== Using Valgrind-3.22.0 and LibVEX; rerun with -h for copyright info
==3308148== Command: ../ipc -s shasum ../data/f2.txt
==3308148== 
shasum(../data/f2.txt) = 'e7d9b82b45d5833c9dada13f2379e7b66c823434  ../data/f2.txt'
==3308148== 
==3308148== HEAP SUMMARY:
==3308148==     in use at exit: 0 bytes in 0 blocks
==3308148==   total heap usage: 3 allocs, 3 frees, 8,548 bytes allocated
==3308148== 
==3308148== All heap blocks were freed -- no leaks are possible
==3308148== 
==3308148== For lists of detected and suppressed errors, rerun with: -s
==3308148== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
