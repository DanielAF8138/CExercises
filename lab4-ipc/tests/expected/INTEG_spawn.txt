shasum(../data/f2.txt) = 'e7d9b82b45d5833c9dada13f2379e7b66c823434  ../data/f2.txt'
